segmenters/nlp/PDFPlumberSegmenter/README.md

```diff
@@ -0,0 +1,46 @@
+# PDFPlumberSegmenter
+
+## 🗝️ Key Concepts
+This is a simple flow test where you can use the PDFExtractorSegmenter.
+This Segmenter will create a chunk per blob.
+
+So if we have a PDF with 2 images and text, we will have 3 chunks; one per each image and one per the text.
+
+
+## 🏃️ Flow
+First you define your flow here:
+    
+    f = Flow().add(uses='PDFExtractorSegmenter', array_in_pb=True)
+
+If you have a PDF with images and text, the ```array_in_pb=True``` is necessary.
```

Comment:
why is this necessary?

Reply:
I haven't updated the readme. This is a copy-paste from another pdf segmenter. 😄